src/Momento.Sdk/TopicClient.cs

```diff
@@ -0,0 +1,84 @@
+#if NETSTANDARD2_0_OR_GREATER
+
+using System;
+using System.Threading.Tasks;
+using Momento.Sdk.Auth;
+using Momento.Sdk.Config;
+using Momento.Sdk.Exceptions;
+using Momento.Sdk.Internal;
+using Momento.Sdk.Responses;
+
+namespace Momento.Sdk;
+
+/// <summary>
+/// Client to perform operations against Momento topics.
+/// </summary>
+public class TopicClient : ITopicClient
+{
+    private readonly ScsTopicClient scsTopicClient;
+    
+    
+    /// <summary>
+    /// Client to perform operations against Momento topics.
+    /// </summary>
+    /// <param name="config">Configuration to use for the transport, retries, middlewares. See <see cref="Configurations"/> for out-of-the-box configuration choices, eg <see cref="Configurations.Laptop.Latest"/></param>
+    /// <param name="authProvider">Momento auth provider.</param>
+    public TopicClient(IConfiguration config, ICredentialProvider authProvider)
```

Comment:
We do not want to use the CacheClient Configuration object here. We need a new TopicConfiguration interface/object with the minimal surface area.